Frontend/src/components/student/NotificationsList.vue

```diff
@@ -0,0 +1,59 @@
+<template>
```

Comment:
Is this on the separate notifications page or the list the shows up when you click the bell?

Reply:
@kraglalbert this is a seperate page that appears when you click the bell